components/Banner/Banner.tsx

```diff
@@ -24,10 +24,9 @@ const InnerWrapper = styled.div`
   height: var(--banner-height);
   display: flex;
   align-items: center;
-  padding-left: 45px;
-  @media ${mobileAndUnder} {
-    max-width: unset;
-    padding: 15px;
+  padding-left: clamp(10px, 45px, 4vw);
```

Comment:
One of my favorite css magic tricks https://developer.mozilla.org/en-US/docs/Web/CSS/clamp